assets/dist/mix-manifest.json

```diff
@@ -11,11 +11,11 @@
     "/scripts/ftnref-convert.js": "/scripts/ftnref-convert.js?id=818f4c429292cb7b6f34057dfcff8b09",
     "/scripts/latex.js": "/scripts/latex.js?id=6393d17f0cdf9131ef0569f5801f1977",
     "/scripts/glossary.js": "/scripts/glossary.js?id=2ff3ebc6fb84ebb5d7c6009d05fcc102",
-    "/scripts/glossary-definition.js": "/scripts/glossary-definition.js?id=e6303e5bdde3c657763391c81f9be4f6",
+    "/scripts/glossary-definition.js": "/scripts/glossary-definition.js?id=1b15b88ec76ae81f1dffb51b9a94d1b1",
     "/scripts/import.js": "/scripts/import.js?id=212e58dacae764405a4408ce7e20c44d",
     "/scripts/login.js": "/scripts/login.js?id=428e5945062c207eabfb8a071122b059",
     "/scripts/network-managers.js": "/scripts/network-managers.js?id=c5acf0a3a6f43e00081189e85a8176e0",
-    "/scripts/organize.js": "/scripts/organize.js?id=22b1775082cd8db5cf4892595eed526c",
+    "/scripts/organize.js": "/scripts/organize.js?id=e49e282f09d8ad872683b175a17f3d37",
```

Comment:
It is strange that changing a single line in a js produced so many changes in other js files after compilation using `npm run build`.
It seems like we do not run this command for a long time and we did not merge this to `dev`. I am not sure if this is the case.

Reply:
This was happening due to an old version of `node`. Fixed after upgrading it.